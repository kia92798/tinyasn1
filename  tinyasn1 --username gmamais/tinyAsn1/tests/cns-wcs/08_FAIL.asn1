MY-MODULE DEFINITIONS AUTOMATIC TAGS::= BEGIN


	MYSQ ::= SEQUENCE { 
		a INTEGER (1..10), 
		b INTEGER OPTIONAL,
		c INTEGER DEFAULT 4
	}

	-- PARTIAL, DEF VAL, OPTIONAL

	MYSQ2 ::= MYSQ ( WITH COMPONENTS { ...,	c OPTIONAL})

	
END
