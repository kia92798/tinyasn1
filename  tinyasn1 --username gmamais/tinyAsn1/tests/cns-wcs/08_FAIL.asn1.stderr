Error Line:12. Presence constraint 'OPTIONAL' specified for component 'c' which is not optional
